fix: update warp route guides to use --symbol instead of --warp (#183)
hyperlane-xyz/v3-docs: docs/guides/remove-trusted-relayer.mdx

When first deploying with the CLI, new warp routes use a Trusted Relayer ISM out of the box so you don't need to run a relayer or validator.

In order to go to production, you'll need to remove this ISM with the [Hyperlane CLI](https://docs.hyperlane.xyz/docs/reference/cli).

## Prerequisites

- The warp route config generated by `hyperlane warp init`.
    - This config is used to deploy a warp route. By default, it takes the filepath of `CURRENT_DIR/configs/warp-route-deployment.yaml`.
- The warp route token symbol used in `hyperlane warp deploy`.
    - The token symbol can be found onchain, or in your local Hyperlane Registry. By default, it is written to `$HOME/.hyperlane/deployments/warp_routes/<tokenSymbol>`.
- Access to the private key that currently owns the warp route.

:::info
If you followed the [Deploy a Warp Route](https://docs.hyperlane.xyz/docs/guides/deploy-warp-route) guide, you most likely have deployed a warp route with a trusted relayer set to a signer address.
:::

> To confirm using the Hyperlane CLI, locate your token symbol and the chain it is deployed on:

Reviewer: Pedantic grammar nit: the chain on which it is deployed

Author: fixed

```bash
hyperlane warp read --symbol <tokenSymbol> --chain <yourchain>
```

> You should be able to select the warp route that you wish to remove the trusted relayer for.

Reviewer: ```suggestion
You can then choose the warp route for which you're removing the trusted relayer.
```
